pallets/fragments/src/mock.rs

```diff
@@ -35,21 +32,31 @@ frame_support::construct_runtime!(
 		ProtosPallet: pallet_protos::{Pallet, Call, Storage, Event<T>},
 		FragmentsPallet: pallet_fragments::{Pallet, Call, Storage, Event<T>},
 		DetachPallet: pallet_detach::{Pallet, Call, Storage, Event<T>},
-		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
-		Assets: pallet_assets::{Pallet, Call, Storage, Event<T>},
-		Accounts: pallet_accounts::{Pallet, Call, Storage, Event<T>},
+		BalancesPallet: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
```

Comment:
Remove Pallet from all, didn't we decide that?

Reply:
yes 😅

Reply:
I renamed it for everything except `FragmentsPallet` - because there also exists a storage item called `Fragments` (which is imported in *test.rs*). 

So if both Fragments (the pallet) and Fragments (the storage item) are imported in tests.rs - it would cause compiler errors